src/poetry/console/commands/plugin/add.py

```diff
@@ -112,9 +112,11 @@ def handle(self) -> int:
                 root_package = ProjectPackage(package.name, package.version)
                 for dependency in package.requires:
                     root_package.add_dependency(dependency)
-
                 break
 
+        if not root_package:
```

Comment:
This change is good, but seems to be unrelated. Can we break it out into a separate PR?

Reply:
yup, just added `type: ignore` comments instead